### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by xuezhen2018** [nycu-acm/DetailSemNet_OSV#1]
May I ask if a pre-trained model can be provided?

**Comment by mcshih** [comment]
@xuezhen2018 Hi, I have already updated the readme. There are model weights for BHSig-B, BHSig-H & CEDAR datasets in [google drive](https://drive.google.com/drive/folders/1NCsdkFgHAbuAIbfeUzJMItvl5wlsJgK8?usp=sharing).

**Comment by YourSonnet18** [comment]
> [@xuezhen2018](https://github.com/xuezhen2018) Hi, I have already updated the readme. There are model weights for BHSig-B, BHSig-H & CEDAR datasets in [google drive](https://drive.google.com/drive/folders/1NCsdkFgHAbuAIbfeUzJMItvl5wlsJgK8?usp=sharing).

May I ask if the pretrained model used the default setting (emd, dis_type, gol_dis, mar_type, etc.)? I saw some lines commented in loss.py.
